lib/smart_answer_flows/marriage-abroad/outcomes/outcome_same_sex_civil_partnership.govspeak.erb

```diff
@@ -42,6 +42,39 @@
                  locals: { calculator: calculator } %>
 
       <%= render partial: 'names_on_documents_must_match.govspeak.erb' %>
+
+    <% elsif calculator.ceremony_country == 'germany' && calculator.partner_is_national_of_ceremony_country? %>
+
+      ##What you need to do
+      <% if calculator.resident_outside_of_uk? %>
+
+        You don’t need a certificate of no impediment (CNI) to prove you’re allowed to marry if you’re a British national resident outside of the UK. However, you’ll need to apply for an exemption (‘Befreiung von der Beibringung des Ehefähigkeitszeugnisses’) from the Higher Regional Court (‘Oberlandesgericht’) in the region where you want to marry.
+
```

Comment:
Similar to the Translation section below, this also appears to be a duplicate of the content in [outcome_opposite_sex_marriage_in_germany](https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_germany.govspeak.erb#L26).

I wonder whether it might be better to re-use the Germany-specific outcome instead of making changes to this one?